### ISSUE TITLE
comfyui extension more features! :smile: 

### ISSUE THREAD

**Issue by cubiq** [tencent-ailab/IP-Adapter#156]
hey @xiaohu2015 it's been a while :smile: I thought you might be interested in the new features of the comfyui extension.

First one is integrated timestepping

![timestepping](https://raw.githubusercontent.com/cubiq/ComfyUI_IPAdapter_plus/main/examples/timestepping.jpg)

Second is compatibility with animations!

https://github.com/tencent-ailab/IP-Adapter/assets/427614/ebab1afe-0b73-4658-b992-49c623a40a44

https://github.com/tencent-ailab/IP-Adapter/assets/427614/2250be86-36c9-4d6a-956e-6dd9435cebca

Congratulations on your amazing models, the best thing that happened to Stable Diffusion :smile: 

Are you going to release any new model soon?

**Comment by cubiq** [comment]
yes, it works, I was wondering if it could be further optimized or if it wouldn't make any difference to train for it

**Comment by xiaohu2015** [comment]
yes, it should be. But compared to LCM, I would like to train ip-adapter with sdxl turbo https://stability.ai/news/stability-ai-sdxl-turbo

**Comment by cubiq** [comment]
that would be amazing!

**Comment by Changlin-Lee** [comment]
> yes, it should be. But compared to LCM, I would like to train ip-adapter with sdxl turbo https://stability.ai/news/stability-ai-sdxl-turbo

Looking forward to seeing it! Any progress on ipadapter with sdxl-turbo?
